packages/__docs__/src/compileMarkdown.tsx

```diff
@@ -39,7 +39,32 @@ import { Playground } from './Playground'
 import { compileAndRenderExample } from './compileAndRenderExample'
 import { Heading } from './Heading'
 import { Link } from './Link'
-import { trimIndent } from './trimIndent'
+
+function trimIndent(str: string) {
+  const lines = `${str.replace(/\r\n/g, '\n').replace(/\r/g, '\n')}\n`.split(
+    '\n'
+  )
+  let indentFound = false
+  let trimmed = ''
+  let indent = ''
+  lines.forEach((line, _i) => {
+    // matches whitespace at the end of a string
+    line.replace(/\s*$/, '')
```

Comment:
I think .replace() does not change the original string, so the whitespace isn't actually removed. The result should be reassigned e.g.: line = line.replace(/\s*$/, '')

Reply:
good catch, fixed